meson.build

```diff
@@ -23,3 +23,8 @@ add_project_arguments(['-Wno-unused-parameter'], language: 'cpp')
 
 subdir('proto')
 subdir('src')
+
+install_data(
+  'wf-osk.desktop',
+  install_dir: '/usr/share/applications'
```

Comment:
Shouldn't this be something like `join_paths(get_option('prefix'), 'share/applications)` ?

Reply:
Because the user may wish to install to other location than /usr

Reply:
And done